internal/install/static_snapshot_yml.go

```diff
@@ -69,4 +69,31 @@ services:
     depends_on:
       package-registry:
         condition: service_healthy
+
+  elastic-agent:
+    image: docker.elastic.co/beats/elastic-agent:${STACK_VERSION}
+    depends_on:
+      elasticsearch:
+        condition: service_healthy
+      kibana:
+        condition: service_healthy
+    healthcheck:
+      test: ["CMD", "grep", "Agent is starting", "/usr/share/elastic-agent/elastic-agent.log"]
```

Comment:
I'm not sure I understand this point. Are you suggesting we tell Agent to write its log files to `/tmp` instead of where it writes them by default and change the health check here somehow?

Reply:
Sorry for the confusion, it's a rather a nice-to-have feature to also observe programmatically logs of elastic-agent, filebeat and metricbeat while testing. Many configuration errors are reported deep down by filebeat or metricbeat's internals.
Logs can be collected in the same single directory and exported as volume.

Reply:
Thanks for explaining. Let's handle this in a follow up PR.